python/ribasim/ribasim/model.py

```diff
@@ -204,7 +204,7 @@ def _apply_crs_function(self, function_name: str, crs: str) -> None:
 
     def node_table(self) -> NodeTable:
         """Compute the full NodeTable from all node types."""
-        df_chunks = [node.node.df.set_crs(self.crs) for node in self._nodes()]  # type: ignore
+        df_chunks = [node.node.df.set_crs(self.crs) for node in self._nodes()]
```

Comment:
CI still seems to disapprove, though I understood locally you saw the opposite? Not sure how to satisfy both.